lib/message_handler.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+
+class Message_Handler
+  attr_reader :message
+
+  def initialize
+    @account_sid = ENV['TWILIO_SID']
+    @auth_token = ENV['TWILIO_AUTH']
+    @client = Twilio::REST::Client.new @account_sid, @auth_token
+  end
+
+  def send_order_message
+    hour_in_secs = 60*60
+    @client.account.messages.create(:body => "Thank you! Your order was placed and will be delivered before #{(Time.now+hour_in_secs).asctime}", :to => "+447913251479", :from => "+441442796217")
```

Comment:
Line is too long. [194/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '+'.